flink-runtime-web/web-dashboard/src/app/share/common/dagre/svg-container.component.ts

```diff
@@ -32,7 +32,8 @@ import * as d3 from 'd3';
 import { select, Selection } from 'd3-selection';
 import { zoom, ZoomBehavior } from 'd3-zoom';
 
-import { SafeAny } from 'interfaces';
+/* eslint-disable-next-line @typescript-eslint/no-explicit-any */
+export type SafeAny = any;
```

Comment:
No way to remove this?

Reply:
We don't need to export this type, though, that I will change.